app/models/invoice.rb

```diff
@@ -115,20 +115,8 @@ def subscription_amount_cents
   end
   alias subscription_amount_currency currency
 
-  def wallet_transaction_amount_cents
```

Comment:
Not sure what happened with this logic after removing it?

Reply:
It is updated directly in https://github.com/getlago/lago-api/pull/983/files#diff-79d4322ca48862bf4414fb50260475427521f6d7ff238b38e6411abc57946714R34, so we have not need to recompute it each time